packages/server-wallet/e2e-test/payer/client.ts

```diff
@@ -101,21 +101,35 @@ export default class PayerClient {
       outbox: [{params}],
     } = await this.time(`update ${channelId}`, async () => this.wallet.updateChannel(channel));
 
+    return (params as WireMessage).data as Payload;
+  }
+
+  public async makePayment(channelId: string): Promise<void> {
+    const payload = await this.createPayment(channelId);
+
     const reply = await this.time(`send message ${channelId}`, async () =>
-      this.messageReceiverAndExpectReply((params as WireMessage).data as Payload)
+      this.messageReceiverAndExpectReply(payload)
     );
 
     await this.time(`push message ${channelId}`, async () => this.wallet.pushMessage(reply));
   }
 
+  public async syncChannel(channelId: string): Promise<void> {
+    const {
+      outbox: [{params}],
+    } = await this.wallet.syncChannel({channelId});
+    const reply = await this.messageReceiverAndExpectReply((params as WireMessage).data as Payload);
```

Comment:
When I began reviewing this PR yesterday, I got to this line, became puzzled, and stopped.

[This](https://github.com/statechannels/statechannels/pull/2573) was merged yesterday. Could it be used to clarify what this line does?

Reply:
I didn't change any behaviour, just copied the pattern from [before](https://github.com/statechannels/statechannels/blob/30bc18df58ede924d39fb49e121a26b68b4dedd4/packages/server-wallet/e2e-test/payer/client.ts#L105)

Reply:
Given that the `server-wallet` now speaks the wire format, I believe that we don't need to (and I assert that we **shouldn't**) copy the pattern from before.

I'm not sure about that though, are you able to clarify?

Reply:
I think, given that the `PaymentManager` is being rewritten, it is safe to ignore my worry here.